code/game/objects/structures/shower.dm

```diff
@@ -64,6 +65,13 @@
 	handle_mist()
 	return TRUE
 
+/obj/machinery/shower/crowbar_act(mob/living/user, obj/item/I)
+	. = ..()
+	to_chat(user, "<span class='notice'>You begin to take apart the [src] with \the [I]...</span>")
+	if(I.use_tool(src, user, 50))
+		new parts_type(loc)
+		user.visible_message("<span class='notice'>[user] takes apart the [src] with \the [I].</span>", "<span class='notice'>You take apart the [src] with \the [I].</span>")
```

Comment:
I always braintoot here but doesn't [src] handle `the`'s? If not they should prob be `\the` in the case of proper nouns

Reply:
it does, I braintooted myself